addons/ballistics/CfgAmmo.hpp

```diff
@@ -6,6 +6,114 @@ class CfgAmmo {
+        submunitionConeType[] = {"poissondisc", 9};
+        submunitionConeAngle = 0.81;
+    };
+    class ACE_12Gauge_Pellets_Submunition_No0_Buck_Deploy: B_12Gauge_Pellets_Submunition_Deploy {
+        airFriction = -0.0033;
+        caliber = 0.5;
+        hit = 2.27;
+    };
+    class ACE_12Gauge_Pellets_Submunition_No1_Buck: B_12Gauge_Pellets_Submunition {
+        caliber = 0.475;
+        submunitionAmmo = "ACE_12Gauge_Pellets_Submunition_No1_Buck_Deploy";
+        submunitionConeType[] = {"poissondisc", 11};
+        submunitionConeAngle = 0.83;
+    };
+    class ACE_12Gauge_Pellets_Submunition_No1_Buck_Deploy: B_12Gauge_Pellets_Submunition_Deploy {
+        airFriction = -0.0038;
+        caliber = 0.475;
+        hit = 1.86;
+    };
+    class ACE_12Gauge_Pellets_Submunition_No2_Buck: B_12Gauge_Pellets_Submunition {
+        caliber = 0.45;
+        submunitionAmmo = "ACE_12Gauge_Pellets_Submunition_No2_Buck_Deploy";
+        submunitionConeType[] = {"poissondisc", 14};
+        submunitionConeAngle = 0.85;
+    };
+    class ACE_12Gauge_Pellets_Submunition_No2_Buck_Deploy: B_12Gauge_Pellets_Submunition_Deploy {
+        airFriction = -0.0048;
+        caliber = 0.45;
+        hit = 1.46;
+    };
+    class ACE_12Gauge_Pellets_Submunition_No3_Buck: B_12Gauge_Pellets_Submunition {
+        caliber = 0.425;
+        submunitionAmmo = "ACE_12Gauge_Pellets_Submunition_No3_Buck_Deploy";
+        submunitionConeType[] = {"poissondisc", 18};
+        submunitionConeAngle = 0.87;
+    };
+    class ACE_12Gauge_Pellets_Submunition_No3_Buck_Deploy: B_12Gauge_Pellets_Submunition_Deploy {
+        airFriction = -0.0067;
+        caliber = 0.425;
+        hit = 1.13;
+    };
+    class ACE_12Gauge_Pellets_Submunition_No4_Buck: B_12Gauge_Pellets_Submunition {
+        caliber = 0.4;
+        submunitionAmmo = "ACE_12Gauge_Pellets_Submunition_No4_Buck_Deploy";
+        submunitionConeType[] = {"poissondisc", 21};
+        submunitionConeAngle = 0.89;
+    };
+    class ACE_12Gauge_Pellets_Submunition_No4_Buck_Deploy: B_12Gauge_Pellets_Submunition_Deploy {
+        airFriction = -0.0085;
+        caliber = 0.4;
+        hit = 0.97;
+    };
+    class ACE_12Gauge_Pellets_Submunition_No4_Bird: B_12Gauge_Pellets_Submunition {
+        caliber = 0.2;
+        hit = 3;
+        submunitionAmmo = "ACE_12Gauge_Pellets_Submunition_No4_Bird_Deploy";
+        submunitionConeType[] = {"poissondisc", 135};
+        submunitionConeAngle = 1.1;
+        triggerSpeedCoef[] = {0.8, 1};
+    };
+    class ACE_12Gauge_Pellets_Submunition_No4_Bird_Deploy: B_12Gauge_Pellets_Submunition_Deploy {
+        caliber = 0.2;
+        airFriction = -0.0800;
+        hit = 0.15;
+    };
+
+    class B_12Gauge_Slug: BulletBase {
+        //caliber = 3; //too high, ~20mm of RHA (450*3*15/1000=20), ~64mm metal
+        caliber = 1.037; //~7mm RHA, ~22.4mm metal, probably still too high though as RHA is hardened.
```

Comment:
Where did you get all these numbers for hit and caliber from? And why not just round this line down to 1.0. It is not like numbers this precise change anything noticably. Decreasing caliber from 3 to 1 also should reduce damage vs. uniforms, vests and helmets a lot. Are we sure we want to touch this at all?

Reply:
For `caliber` I used an online armour penetration calculator to figure out the real world penetration of all the different projectiles and then used BI's equation `velocity[m/s] * caliber * penetrability / 1000` to convert those to config values, with `velocity` taken from the config values of the magazines and `penetrability` taken from config values in BI's material files.

For `hit` I took the real world muzzle energy of 9mm, compared it to the hit of 9mm in game and then used that to calculate the hit values of various shot types based on their respective real world muzzle energies.

Reply:
That equation is missing the equal sign?

I don't think this is a good method to determine hit.

Reply:
`Penetration depth in mm = velocity[m/s] * caliber * penetrability / 1000`
Penetrability for RHA steel is 15, for concrete 80, for meat 250.
So `450[m/s] * 1.037[caliber] * 15[penetrability] / 1000 = 6.99975 mm` penetration of rolled homogeneous armour, a type of hardened steel armour plate.


For hit, I think my method worked well enough and during in game testing of the different shot types it felt good. I'm open to suggestions for a different method though.

Reply:
Vanilla values have way less decimal places:
```cpp
    class B_9x21_Ball: BulletBase {
        hit = 5;
        caliber = 1.2;
    };
```
This PR just fools accuracy to this stuff that does not exist. I am not opposed to merging this (if formatting is ok), but I won't approve such changes as I don't want to bear the onus.